src/api/spec/controllers/webui/main_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe Webui::MainController do
+  let(:user) { create(:confirmed_user) }
```

Comment:
I have to talk to henne about this. But my understanding would be that we don't plan to swap such settings randomly. So probably this is intentional and how we want our test suite be set up.